OrbitCore/LinuxTracingHandler.cpp

```diff
@@ -159,7 +159,7 @@ void LinuxTracingHandler::OnGpuJob(
   timer_user_to_sched.m_End = gpu_job.GetAmdgpuSchedRunJobTimeNs();
   timer_user_to_sched.m_Depth = gpu_job.GetDepth();
 
-  constexpr const char* sw_queue = "sw queue";
+  const std::string sw_queue("sw queue");
```

Comment:
what was the bug here?

Reply:
Sorry, this was a bad attempt at fixing something, but this was not the issue. Reverted to using constexpr const char*.